schema.sql

```diff
@@ -22,8 +22,40 @@ CREATE TABLE species (
 
```

Comment:
### Optional
- [ ] It would be better to write more commit messages, this helps us track part of our code and let us also try and capitalize all our commit messages

Reply:
Thank you for that